tags:: [[Base Sequence]], [[DNA]], [[Bacterial]], [[DNA-Directed RNA Polymerases]], [[Escherichia coli]], [[Genes]], [[Lac Operon]], [[Models]], [[Genetic]], [[Mutation]], [[Promoter Regions]], [[Sigma Factor]], [[Transcription Factors]], [[Transcription]]
date:: [[Apr 20th, 1989]]
issn:: 0022-2836
issue:: 4
extra:: PMID: 2661828
doi:: 10.1016/0022-2836(89)90568-8
title:: Altered promoter recognition by mutant forms of the sigma 70 subunit of Escherichia coli RNA polymerase
pages:: 591-603
volume:: 206
item-type:: [[journalArticle]]
original-title:: Altered promoter recognition by mutant forms of the sigma 70 subunit of Escherichia coli RNA polymerase
language:: eng
publication-title:: Journal of Molecular Biology
journal-abbreviation:: J Mol Biol
authors:: [[D. A. Siegele]], [[J. C. Hu]], [[W. A. Walter]], [[C. A. Gross]]
library-catalog:: PubMed
links:: [Local library](zotero://select/library/items/9G95AD5Q), [Web library](https://www.zotero.org/users/6106196/items/9G95AD5Q)

- [[Abstract]]
	- We have systematically assayed the in vivo promoter recognition properties of 13 mutations in rpoD, the gene that encodes the sigma 70 subunit of Escherichia coli RNA polymerase holoenzyme, using transcriptional fusions to 37 mutant and wild-type promoters. We found three classes of rpoD mutations: (1) mutations that suggest contacts between amino acid side-chains of sigma 70 and specific bases in the promoter; (2) mutations that appear to affect either sequence independent contacts to promoter DNA or isomerization of the polymerase; and (3) mutations that have little or no effect on promoter recognition. Our results lead us to suggest that a sequence near the C terminus of sigma 70, which is similar to the helix-turn-helix DNA binding motif of phage and bacterial DNA binding proteins, is responsible for recognition of the -35 region, and that a sequence internal to sigma 70, in a region which is highly conserved among sigma factors, recognizes the -10 region of the promoter. rpoD mutations that lie in the recognition helix of the proposed helix-turn-helix motif affect interactions with specific bases in the -35 region, while mutations in the upstream helix, which is thought to contact the phosphate backbone, have sequence-independent effect on promoter recognition.
- Attachments
	- [PubMed entry](http://www.ncbi.nlm.nih.gov/pubmed/2661828)
	- [PDF](zotero://select/library/items/QT4PZVA5) {{zotero-imported-file QT4PZVA5, "Siegele et al. - 1989 - Altered promoter recognition by mutant forms of the sigma 70 subunit of Escherichia coli RNA polymer.pdf"}}
- Sigma factors are small subunits of RNA polymerase that preferentially bind to certain promoter sequences, increasing the transcription of those genes
	- from sam
- \sigma subunit of RNA pol is what confers specificity to promoter regions
	- \sigma70 is the main one that's most commonly used
	- Bacteria can turn on other \sigma factors in response to stresses to change gene expression
- They use transcriptional fusions to mutant and wild type promoters with mutants of RNA pol \sigma70 to see what the mutations are doing
- \sigma70 doesn't bind DNA
	- Maybe it can recognize the conserved sequences and the distance between them as a stabilizing factor?
	- Sit on top of DNA and not bind with any strength until it has that -10 and -35 site?
- Introduction
	- Several steps in RNA synthesis
		- binding of RNA polymerase to promoter DNA to form a closed complex
		- at least two isomerizations of the protein-DNA complex to form an open complex
			- DNA at transcription start site is melted
		- Formation of initial RNA bonds
		-