file:: [Obholzer et al. - 2012 - Rapid positional cloning of zebrafish mutations by linkage and homozygosity mapping using whole-geno.pdf](file://C:/Users/David/Zotero/storage/LSPPKD7C/Obholzer et al. - 2012 - Rapid positional cloning of zebrafish mutations by linkage and homozygosity mapping using whole-geno.pdf)
file-path:: file://C:/Users/David/Zotero/storage/LSPPKD7C/Obholzer et al. - 2012 - Rapid positional cloning of zebrafish mutations by linkage and homozygosity mapping using whole-geno.pdf

- escribe an approach for positional cloning by wholegenome sequ
  ls-type:: annotation
  hl-page:: 2
  hl-color:: yellow
  id:: 69118cfd-69fb-476c-a98c-3a2395f394f1
- omozygosity mapp
  ls-type:: annotation
  hl-page:: 2
  hl-color:: red
  id:: 69118d1d-1044-4847-bb7a-3ba4ac7738d2
- ring previously known single nucleotide polymorph
  ls-type:: annotation
  hl-page:: 2
  hl-color:: yellow
  id:: 69118d2a-bd60-4f08-9446-64cd2a99bad5
- raction of total homozygous over total SNPs per chromosome determined the candidate chromoso
  ls-type:: annotation
  hl-page:: 2
  hl-color:: red
  id:: 69118db4-fd1d-4ada-baef-c66e30e147ad
  hl-stamp:: 1762758071230
- ess foolproof depending on the extent of consanguinity in the fis
  ls-type:: annotation
  hl-page:: 3
  hl-color:: yellow
  id:: 69118e3a-59c4-494e-8a28-684a4a3d41ff
- adults and identified mutant carriers (m/+) by in-crossing and checking for phenotypically mutant progeny(m/m). We in-crossed F1 heterozygotes to generate F2 clutches, which we sorted by phenotype into wild-type and mutant pools to collect at least 200 mutant embry
  ls-type:: annotation
  hl-page:: 3
  hl-color:: red
  id:: 69118e52-6468-4e09-a7dc-bfc8f84ea7a3
  hl-stamp:: 1762758227484
- ing through a larger candidate region can be done efficiently using bioinforma
  ls-type:: annotation
  hl-page:: 3
  hl-color:: yellow
  id:: 69118e69-bd8d-494c-86ee-29215e80907c
- stead, we collected mutants from approximately eight pairs of carriers that had been maintained through outcrosses to prevent low fecun
  ls-type:: annotation
  hl-page:: 3
  hl-color:: red
  id:: 69118e74-45a9-4339-99b0-faf73493f88c
  hl-stamp:: 1762758263150
- alculated and plotted the allele frequency of the two contributing haplot
  ls-type:: annotation
  hl-page:: 4
  hl-color:: yellow
  id:: 691190e6-f96d-424b-9ff6-62e9bddb68b2
- neral, we found that the chromosome with the greatest proportion of mutagenic strain SNPs (AB) reliably identifies the correct chromoso
  ls-type:: annotation
  hl-page:: 4
  hl-color:: yellow
  id:: 691190f3-5798-45d0-ad0b-fff5e981f956
- single candidate lesion in lmx1b (ENSDARG00000068365) that introduced a premature termination codon at 34,129,111 on chromosome 8, only 871 kb from the midpoint of the compromise fitâ€™s critical regio
  ls-type:: annotation
  hl-page:: 4
  hl-color:: green
  id:: 6911912a-29f8-401e-a938-61de938598f8
- terozygous SNP density analyses and the presence of a sizable intrastrain SNP variation suggested that mapping could be performed without defined haplotype linkag
  ls-type:: annotation
  hl-page:: 4
  hl-color:: yellow
  id:: 69119136-b620-4aac-afa0-20c3468c1879
  hl-stamp:: 1762758967966
- ntify the putative chromosome, we calculated the fraction of homozygous SNPs out of total SNPs per chromoso
  ls-type:: annotation
  hl-page:: 5
  hl-color:: yellow
  id:: 6911916b-ca5f-47c9-a4d6-b323f7edc190
- d chose the chromosome with the largest value, chromosome 13, as the candidate for harboring the mutatio
  ls-type:: annotation
  hl-page:: 5
  hl-color:: red
  id:: 69119178-a9c9-438a-8e6a-626b79290951
- ose the bin with the peak homozygous to heterozygous ratio at position 46,419,377 on the candidate chromosome 13 as the SNP density mapping predic
  ls-type:: annotation
  hl-page:: 5
  hl-color:: yellow
  id:: 69119183-e82a-42fa-bec5-48f2fd16a559
- nformatics pipeline again identified a single candidate in the critical interval at 43,745,899, effecting the essential invariant intronic portion of an acceptor splice site of cadherin 23
  ls-type:: annotation
  hl-page:: 6
  hl-color:: green
  id:: 69119193-47e3-4a73-88e8-09f06a98390f
- duce the number of candidates, we filtered out all other wild-type SNPs in our zebrafish SNP databas
  ls-type:: annotation
  hl-page:: 7
  hl-color:: red
  id:: 69119256-5dd7-40c0-a473-5ca9224e2304
- as shown below, further evidence supported this single candidate as being the correct prediction in all four case
  ls-type:: annotation
  hl-page:: 7
  hl-color:: green
  id:: 6911925c-9d91-4911-aaae-0231efb2e095
- zebrafish SNP univers
  ls-type:: annotation
  hl-page:: 7
  hl-color:: green
  id:: 6911926a-43f7-4c24-b72f-2285b228a4f7