date:: [[Mar 2nd, 2007]]
issue:: 5816
extra:: Publisher: American Association for the Advancement of Science
doi:: 10.1126/science.1138527
title:: Reconstitution of DNA Segregation Driven by Assembly of a Prokaryotic Actin Homolog
pages:: 1270-1274
volume:: 315
item-type:: [[journalArticle]]
access-date:: 2025-09-22T17:58:48Z
original-title:: Reconstitution of DNA Segregation Driven by Assembly of a Prokaryotic Actin Homolog
url:: https://www.science.org/doi/10.1126/science.1138527
publication-title:: Science
authors:: [[Ethan C. Garner]], [[Christopher S. Campbell]], [[Douglas B. Weibel]], [[R. Dyche Mullins]]
library-catalog:: science.org (Atypon)
links:: [Local library](zotero://select/library/items/PAE8NT4X), [Web library](https://www.zotero.org/users/6106196/items/PAE8NT4X)

- [[Abstract]]
	- Multiple unrelated polymer systems have evolved to partition DNA molecules between daughter cells at division. To better understand polymer-driven DNA segregation, we reconstituted the three-component segregation system of the R1 plasmid from purified components. We found that the ParR/parC complex can construct a simple bipolar spindle by binding the ends of ParM filaments, inhibiting dynamic instability, and acting as a ratchet permitting incorporation of new monomers and riding on the elongating filament ends. Under steady-state conditions, the dynamic instability of unattached ParM filaments provides the energy required to drive DNA segregation.
- Attachments
	- [Full Text PDF](https://www.science.org/doi/pdf/10.1126/science.1138527) {{zotero-imported-file 3FJUAIP3, "Garner et al. - 2007 - Reconstitution of DNA Segregation Driven by Assembly of a Prokaryotic Actin Homolog.pdf"}}
- Polymer driver DNA segregation
- Reconstituted R1 plasmid system in vitro from purified protein
- ((68d20634-9bac-4b3c-8639-0dd0efc9006e))
- Eukaryotes use tubulin polymerization to drive alignment and segregation
	- eubacteria uses actin-like filaments
	- some have their own elements, such as R1
- R1
	- 100kb low copy number plasmid
	- has the par operon which does segregation machinery
	- parM - actin like protein that assemble into dynamically unstable filaments
	- parR protein binds cooperatively to sequence repeats in parC (gene)
		- forms a complex that binds ParM filaments
- ParM
	- similar to actin but 3 important differences
	- rapid spontaneous nucleation
	- symmetrical elongation (bidirectional)
	- dynamic instability
		- like microtubules
	- all together this means that the filaments are short and unstable (1.5 \mu\m and 20s lifetime)
- 350nm beads with fluorescent DNA containing parC sequence
	- add ParR to this and fluorescent ParM + ATP
- Beads get covered in a radial array that looks like a star
	- dynamic
	- needs ATP, ParR, and parC DNA sequence
	- non hydrolysable ATP analog AMP-PNP results in much larger growth
		- dynamic instability is limiting the length, like in microtubules
	- ((68d20859-d64e-4350-81b6-48d0821566dd))
		- It's way bigger with AMPPNP (normal has a 3\mu\m limit)
- ParM filaments also connect further apart beads with more stability
	- connected beads move together in fluid flow -> strong attachment
	- both ends of the filament can interact with the parC region
		- not like they meet in the middle or something
	- Bundle elongates at a constant rate
		- pushing apart the beads
		- goes over long distances >120\mu\m
	- Can have multiple ParM filaments per bead and have them make a polygon
		- this expands equilaterally
- Search and capture process
	- unconnected ParM asters come close to each other they make a filament and start pushing apart
	- If you cut the filament it depolymerizes
		- you need bipolar stabilization
	- This explains why ParM spindles happen only in cells with two or more plasmids
		- need to have two parC regions to attach to
	- Brownian ratchet mechanism
		- orientation doesn't need cellular landmarks
		- it just goes until it encounters resistance
- Where does the new protein add?
	- photobleach two spots as reference
		- as the spindle grows, they do not move apart, but both get further from the bead
		- ((68d20cb2-1891-40cd-82fd-4e6c1f513cf1))
		- no recovery of fluorescence in the bleached zone
			- polymerization is not in the middle and there is no sliding
	- Monomers incorporate exclusively at the surface of parC beads
		- like actin
	- rate constant is the same as in free filament polymerization
- for free filaments steady state monomer concentration is 2.3 \mu\M
	- ATP critical concentration is 0.6 \mu\M, and that's the lower threshold with parC stabilization
	- Can get less than 2.3 but not less than 0.6
	- unattached ParM filaments are still cycling and that is driving movement
- ((68d20e6c-2b41-4bbf-86d1-b71bf1442e60))
-