date:: [[Jan 7th, 1994]]
issn:: "0036-8075, 1095-9203"
issue:: 5143
doi:: 10.1126/science.8272867
title:: Target of the Transcriptional Activation Function of Phage λ cl Protein
pages:: 75-77
volume:: 263
item-type:: [[journalArticle]]
access-date:: 2025-09-07T20:42:37Z
original-title:: Target of the Transcriptional Activation Function of Phage λ cl Protein
language:: en
url:: https://www.science.org/doi/10.1126/science.8272867
publication-title:: Science
journal-abbreviation:: Science
authors:: [[Mei Li]], [[Henry Moyle]], [[Miriam M. Susskind]]
library-catalog:: DOI.org (Crossref)
links:: [Local library](zotero://select/library/items/GS3NEL75), [Web library](https://www.zotero.org/users/6106196/items/GS3NEL75)

- [[Abstract]]
	- Activation of transcription initiation by the cI protein of phage λ is thought to be mediated by a direct interaction between cI and RNA polymerase at the
	                P
	                RM
	                promoter. Two negatively charged amino acid residues in the DNA binding domain of cI play a key role in activation, suggesting that these residues contact RNA polymerase. The subunit of RNA polymerase involved was identified by selecting polymerase mutants that restored the activation function of a mutant form of cI protein. Although previous studies suggest that several activators interact with the α subunit of RNA polymerase, the results here suggest that cI interacts with the σ subunit. An arginine to histidine change near the carboxyl terminus of σ specifically suppresses an aspartic acid to asparagine change in the activation region of cI. This finding supports the direct-contact model and suggests that a cluster of positively charged residues near the carboxyl terminus of σ is the target of the negatively charged activation region of cI.
- Attachments
	- [PDF](zotero://select/library/items/ZZD44UN4) {{zotero-imported-file ZZD44UN4, "Li et al. - 1994 - Target of the Transcriptional Activation Function of Phage λ cl Protein.pdf"}}
- They think it interacts with \sigma
- pc stands for positive control
	- these mutants change AAs in the first \alpha helix of the DNA binding motif
- \sigma70 is predicted to be close to c1, but not known if they interact
	- this is because sigma recognizes the -35 region which is right next to c1 binding site
- mutate RNA pol so that pc mutants can still activate it
	- this is a common strategy
- Methods
	- strain carrying two P22 prophages
		- kan resistance under control of wt P_rm
		- second prophage has \lambda c1 gene at low level from the lac promoter
			- fusion is in a dispensable region of phage genome
			- this gene is a pc mutant and can't really activate the wt P_rm
		- so strain is sensitive to kan
	- mutagenize plasmids with rpoA or rpoD
		- select for kan resistance
- only rpoD mutants were found to give kan resistance, showing that it interacts with \sigma
	- and they're actually all the same mutant, RH596
	- Same one found by [[D. A. Siegele]] in [[Altered promoter recognition by mutant forms of the sigma 70 subunit of Escherichia coli RNA polymerase]]
		- not super sure why this is the same mutant
		- I guess maybe araC binds in a similar mechanism?
- allele specific
	- ((68e354c2-5693-42f7-a958-3395503916be))
	- only that one mutant is fixed by the rpoD mutation
- t